docs/tutorials/app-examples.rst

```diff
@@ -1,10 +1,9 @@
+.. _app-examples:
+
 Application examples
 ####################
 
-Existing applications
-=====================
-
-* :github:`Kinto React Boilerplate <Kinto/kinto-react-boilerplate>`: A starter kit for JavaScript application.
-* :github:`Routina <leplatrem/Routina>`: An offline-first demo Web app.
```

Comment:
I first wanted to keep a list of *featured apps*, but then thought it could conceal the wiki page...